lib/registration.pm

```diff
@@ -145,12 +145,19 @@ sub scc_version {
 Wrapper for SUSEConnect -p $name.
 =cut
 sub add_suseconnect_product {
-    my ($name, $version, $arch, $params, $timeout) = @_;
+    my ($name, $version, $arch, $params, $timeout, $retry) = @_;
     assert_script_run 'source /etc/os-release';
     $version //= '${VERSION_ID}';
     $arch    //= '${CPU}';
     $params  //= '';
-    assert_script_run("SUSEConnect -p $name/$version/$arch $params", $timeout);
+    $timeout //= 300;
+    $retry   //= 1;                 # run SUSEConnect a 2nd time to workaround the gpg error due to missing repo key on 1st run
```

Comment:
This will change the behavior this function had. The default should be 0, to not affect registering other modules.